Thoroughly updated Sample Databases page with links, etc.

douglaslMS opened the pull request:
## Summary

This page was horribly out of date, with incorrect branding and old links. I updated all.

Fixes #7590

douglaslMS committed:
Thoroughly updated Sample Databases page with links, etc.

douglaslMS committed:
Reviewed first build and cleaned up.

douglaslMS committed:
More cleanup.

mairaw commented:
This topic has been on my radar because it has high pageviews and low CSAT. From my conversation with the SQL team in the past, it seems that we wanted to push folks towards https://github.com/Microsoft/sql-server-samples.

Maybe we can keep Northwind instructions for historical reasons (that link takes you to a SQL 2000 version of Northwind) and because a lot of pages link to this one from our current outdated topics, but I think we need to create an issue to refresh topics that use SQL Server to start using the recommended database instead of Pubs, Northwind or AdventureWorks. We could even ask for help from the community and prioritize based on page views.

mairaw commented on docs/framework/data/adonet/sql/linq/downloading-sample-databases.md:
```diff
@@ -1,50 +1,48 @@
-#### To download Management Studio Express  
-  
--   Follow the instructions at the [SSMSE site](https://go.microsoft.com/fwlink/?LinkId=95933).  
+-   Follow the instructions at the [Download SQL Server Management Studio (SSMS)](https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms) page.  
```
link should be /sql/ssms/download-sql-server-management-studio-ssms, otherwise it doesn't work in the VS offline books

douglaslMS replied:
I have just fixed this in a new commit.

mairaw reviewed (changes requested):
I think we need to start pushing folks to the newer samples location and maybe keep instructions for NorthWind while we're still using that database in our content. But a plan to refresh those should be created.

douglaslMS commented:
@mairaw I will fix the link. If our code samples still use Northwind, however, we can't send them to the sql-server-samples repo at this point, since Northwind and Pubs aren't available there.

douglaslMS committed:
Fixed link to SQL repo per reviewer comment.

douglaslMS commented:
@mairaw How do I clear the "Changes requested" flag, now that I've fixed the link format in a separate commit?

douglaslMS commented:
@mairaw @rpetrusha I've never had to address "Changes requested" before this in a PR. I made Maira's suggested fix in a separate commit. How do I clear the "Changes requested" flag, now that I've fixed the link format in a separate commit?

mairaw commented:
I need to re-review @douglaslMS. But why addressing the feedback on a separate PR?

mairaw commented:
We also usually add a label saying changes-addressed so we can tell the feedback has been addressed.

douglaslMS commented:
@mairaw I addressed your feedback in a separate commit to this PR, not in a separate PR. The link format should now be what you requested for compatibility with offline books.

douglaslMS commented:
@JennieHubbard cc @craigg-msft These are the pending updates that you're looking for.

mairaw commented:
I'm sorry for the delay. I've been busy reviewing other PRs this morning, but I don't see my feedback addressed. Are you planning to do further changes later?

mairaw commented:
Also, feedback indicates that people were looking for AdventureWorks info on this page. We might be sending them from pages where AdventureWorks DB is used?

mairaw commented on docs/framework/data/adonet/sql/linq/downloading-sample-databases.md:
```diff
@@ -1,50 +1,48 @@
+
+## Download the Sample Databases
+
+### To download the sample databases
```
if you don't add other databases here, you should just remove this redundant heading or add an intro paragraph
